### DISCUSSION TITLE
[Unhandled promise rejection: TypeError: undefined is not an object (evaluating '_supabase.supabase.auth')]

### DISCUSSION THREAD

**Discussion by HassanIyan** [supabase/supabase#3772] [Questions]
i am developing a app using expo and supabase...
and while sign up it give this error:

[Unhandled promise rejection: TypeError: undefined is not an object (evaluating '_supabase.supabase.auth')]
at node_modules/react-native/Libraries/Pressability/Pressability.js:697:17 in _performTransitionSideEffects
at node_modules/react-native/Libraries/Pressability/Pressability.js:634:6 in _receiveSignal
at node_modules/react-native/Libraries/Pressability/Pressability.js:528:8 in responderEventHandlers.onResponderRelease

here is my code: 
async function handleSignup() {   
        let { error } = await supabase.auth.signUp({
            email: email,
            password: password
        })
        if (error) {
            console.log(error)
        }
}

**Comment by GaryAustin1** [comment]
I'm not sure if your error is related or not, but an issue came up recently with expo...
supabase/supabase-js#295
Might be worth checking out.

**Reply by HassanIyan** [reply]
just like said in that i have added detectSessionInUrl: false to the code...

**Reply by GaryAustin1** [reply]
If that worked for you can you check answered, this is the 2nd time this has come up this week.

**Reply by HassanIyan** [reply]
but, after that this error started coming when signup function has been called...

[Unhandled promise rejection: TypeError: undefined is not an object (evaluating '_supabase.supabase.auth')]
at node_modules/react-native/Libraries/Pressability/Pressability.js:697:17 in _performTransitionSideEffects
at node_modules/react-native/Libraries/Pressability/Pressability.js:634:6 in _receiveSignal
at node_modules/react-native/Libraries/Pressability/Pressability.js:528:8 in responderEventHandlers.onResponderRelease

**Reply by GaryAustin1** [reply]
Unfortunately, I just happened to see there was an Expo issue a couple of days ago and linked a couple of people to that issue (one was clearly a match to that issue).  I don't use Expo.  Obviously don't mark as answered if it did not solve your problem, or if you have a clearly different issue then ask a new question.

**Reply by GaryAustin1** [reply]
LOL, I did realize you were same poster on both.  So sorry.
